charts/t8s-cluster/templates/workload-cluster/cloud-controller-manager.yaml

```diff
@@ -41,14 +41,33 @@ spec:
       valuesKey: cloud.conf
     - <<: *valuesFrom
       targetPath: cloudConfig.metadata.hash
+  {{- if .Values.controlPlane.hosted }}
+  postRenderers:
+    - kustomize:
+        patches:
+          - target:
+              kind: DaemonSet
+            patch: |
+              - op: replace
+                path: /kind
+                value: Deployment
```

Comment:
Why make the Daemonset a deployment?

Reply:
Because we don't have anything to run the cloud-controller-manager on, especially not a control-plane

Therefore it needs to run on the compute-plane and doesn't need to run on every compute node

Reply:
Alrighty